test/e2e/csv_e2e_test.go

```diff
@@ -3465,44 +3376,47 @@ func waitForDeployment(c operatorclient.ClientInterface, name string) error {
 	})
 }
 
-func waitForDeploymentToDelete(t GinkgoTInterface, c operatorclient.ClientInterface, name string) error {
-	return wait.Poll(pollInterval, pollDuration, func() (bool, error) {
-		t.Logf("waiting for deployment %s to delete", name)
+func waitForDeploymentToDelete(c operatorclient.ClientInterface, name string) error {
+	var err error
+
+	Eventually(func() (bool, error) {
+		ctx.Ctx().Logf("waiting for deployment %s to delete", name)
 		_, err := c.GetDeployment(testNamespace, name)
 		if errors.IsNotFound(err) {
-			t.Logf("deleted %s", name)
+			ctx.Ctx().Logf("deleted %s", name)
 			return true, nil
 		}
 		if err != nil {
-			t.Logf("err trying to delete %s: %s", name, err)
+			ctx.Ctx().Logf("err trying to delete %s: %s", name, err)
 			return false, err
 		}
 		return false, nil
-	})
+	}).Should(BeTrue())
+	return err
 }
 
-func waitForCSVToDelete(t GinkgoTInterface, c versioned.Interface, name string) error {
+func waitForCSVToDelete(c versioned.Interface, name string) error {
 	var err error
 
-	err = wait.Poll(pollInterval, pollDuration, func() (bool, error) {
+	Eventually(func() (bool, error) {
 		fetched, err := c.OperatorsV1alpha1().ClusterServiceVersions(testNamespace).Get(context.TODO(), name, metav1.GetOptions{})
 		if errors.IsNotFound(err) {
 			return true, nil
 		}
-		t.Logf("%s (%s): %s", fetched.Status.Phase, fetched.Status.Reason, fetched.Status.Message)
+		ctx.Ctx().Logf("%s (%s): %s", fetched.Status.Phase, fetched.Status.Reason, fetched.Status.Message)
 		if err != nil {
 			return false, err
 		}
 		return false, nil
-	})
+	}).Should(BeTrue())
 
 	return err
 }
 
-func waitForNotFound(t GinkgoTInterface, getFunction func() error) error {
+func waitForNotFound(getFunction func() error) error {
```

Comment:
The same basic comment from `waitForCSVToDelete` applies here. They're nearly the same function, but this one is apparently only called from a single place. Can we delete this function altogether and use Eventually directly at its one callsite?